src/main/java/dk/cngroup/lentils/entity/Cypher.java

```diff
@@ -53,6 +53,12 @@
     @NotEmpty(message = "Adresa mapy nesmí být prázdná.")
     private String mapAddress;
 
+    @ManyToMany(cascade = CascadeType.DETACH, fetch = FetchType.EAGER)
+    @JoinTable(name = "cypher_organizers",
+            joinColumns = @JoinColumn(name = "cypher_id"),
+            inverseJoinColumns = @JoinColumn(name = "user_id"))
+    private List<User> organizers;
```

Comment:
Bude to chtít test, že jde smazat šifru i když má přirazené organizátory, pokud tam takový nemáš. Díval jsem se, ale možná jsem si nevšiml.

Reply:
Ok, šifra z UI smazat jde, ale když se ti podaří test napsat, bude to za 1000 bodů.

Reply:
Myslím že to mám na lokálu už udělané, ale nestihl jsem commit jak jsem spěchal na bus :D 🥇